less/index.less

```diff
@@ -49,11 +49,192 @@ table {
 }
 
 /* Set every element's box-sizing to border-box */
+// Them Variables
+@main-color: @broom;
+@accent-color: @royalblue;
+
+// Colors Only
+
+@broom: #FFF727;
+@royalblue: #5A31ED;
+
+// Font Stack
+@font-stack: Arial, 'sans-serif';
+
+// escaping
+@tablet: ( max-width: 800px );
+@mobile: ( max-width: 600px );
+
+
+// General Styles
+
 * {
     box-sizing: border-box;
 }
 
+html {
+	font-size: 62.5%;
+}
+
 html, body {
-    height: 100%;
+		height: 100%;
+		background-color: #F6F5F9;
+}
+
+h1, h2, h3, h4, h5 {
+	font-size: 2rem;
+	color: black;
+}
+
+img {
+	width:80%;
+	height: auto;
+	margin: 0rem;
+}
+
+p {
+	line-height: 1.25;
+	margin: 20px 0;
+	font-size: 1.8rem;
+}
+
+.container {
+	max-width: 800px;
+	margin: 0 auto;
+	font-size: 1.6rem;
+}
+
+header {
+border: 1px solid black;
+background-color: lightgrey;
+margin-left: 0;
+padding: 20px 0;
+display: flex;
+justify-content: space-between;
+align-items: center;
+
+	@media @mobile {
+		flex-direction: column;
+		align-items: center;
+	}
+	h1 {
+		margin-left: 20px;
+		margin-bottom: 10px;
+		font-size: 4rem;
+	}
+	nav {
+		margin-left: 40px;
+
+			a {
+				text-decoration: none;
+				color: grey;
+				margin: 20px;
+				font-size: 1.8;
+			}
+		}
+}
+
+.upper-content {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-evenly;
+	align-content: center;
+	margin: 2rem;
+
```

Comment:
for upper-content I would do a media query for mobile that applies flex-direction: column; and align-items: center to stack everything up, it's running out of room as it is here. when you do use align you'll be using align-items most of the time, align-content is a weird one